src/components/ExportCSV/ExportCSV.jsx

```diff
@@ -0,0 +1,326 @@
+            <p className="option-header">Filter Options</p>
+            <div className="filters">
+              <div className="filter-options">
+                <p className="filter-names">Boxes</p>
+                <div className="drop-option">
+                  <Select
+                    fontSize="20px"
+                    width="300px"
+                    value={boxes}
+                    onChange={e => setBoxes(e.target.value)}
+                  >
+                    <option value="all">All</option>
+                    <option value="custom">Custom</option>
+                  </Select>
+                  {boxes === 'custom' && (
+                    <div>
+                      <Input
+                        value={boxesCustom}
+                        className="custom-text"
+                        placeholder="e.g. 1-9, 6, 12"
+                        isInvalid={checkRange(boxesCustom)}
+                        onChange={e => setBoxesCustom(e.target.value)}
+                      />
+                      {checkRange(boxesCustom) && (
+                        <p className="error-message">Invalid page range, e.g. 1-9, 6, 12</p>
+                      )}
+                    </div>
+                  )}
+                </div>
+              </div>
+              <div style={{ display: 'flex', flexDirection: 'column' }}>
+                <div className="filter-options">
+                  <p className="filter-names">Date</p>
+                  <div className="drop-option">
+                    <Select
+                      fontSize="20px"
+                      width="300px"
+                      value={date}
+                      onChange={e => setDate(e.target.value)}
+                    >
+                      <option value="all">All</option>
+                      <option value="single">Single Day</option>
+                      <option value="range">Range</option>
+                    </Select>
+                    {date === 'single' && (
+                      <div>
+                        <Input
+                          value={dateCustom}
+                          className="custom-text"
+                          placeholder="MM/DD/YY"
+                          isInvalid={checkDate(dateCustom)}
+                          onChange={e => setDateCustom(e.target.value)}
+                        />
+                        {checkDate(dateCustom) && (
+                          <p className="error-message">Invalid date, use MM/DD/YY format</p>
+                        )}
+                      </div>
+                    )}
+                    {date === 'range' && (
+                      <div>
+                        <div className="custom-range-boxes">
+                          <div>
+                            <FormLabel fontSize="16px" className="date-label" htmlFor="from-date">
+                              Start
+                            </FormLabel>
+                            <Input
```

Comment:
Same here, we can use React's package for date ranges: https://yarnpkg.com/package/react-dates. With this, you won't need to validate the date range or check if end date is before start date (but be sure to test this lol).